lib/benchee/benchmark/measure/memory.ex

```diff
@@ -63,14 +63,18 @@ defmodule Benchee.Benchmark.Measure.Memory do
       :ready_to_begin -> nil
     end
 
-    result = fun.()
-    send(parent, {:result, result})
+    send(parent, fun.())
```

Comment:
I like this! It's a very pragmatic solution. Maybe a comment in the receive further up could help though?

And while I like the conciseness, I think part of me likes:

```
result = fun.()
send(parent, result)
```

a tiny bit better... just to make it clear that we're sending the result of the function being evaluated there. Although, it should be painfully clear... huh. Not sure. Looks odd to call the function here a bit but that might just be me - your call :)